res_partner_operating_unit/tests/test_res_partner_operating_unit.py

```diff
@@ -1,57 +1,77 @@
 # © 2017 Niaga Solution - Edi Santoso <repodevs@gmail.com>
 # Copyright (C) 2020 Serpent Consulting Services
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
-from odoo.tests import common
+from odoo import Command
+from odoo.exceptions import UserError
+from odoo.tests.common import TransactionCase
 
 
-class TestResPartnerOperatingUnit(common.TransactionCase):
-    def setUp(self):
-        super(TestResPartnerOperatingUnit, self).setUp()
-        self.res_partner_model = self.env["res.partner"]
-        self.res_users_model = self.env["res.users"]
+class TestResPartnerOperatingUnit(TransactionCase):
```

Comment:
`TransactionCase` can be replaced with `OperatingUnitCommon`

And the whole unit test can be simplified since a lot of common things are declared in the `OperatingUnitCommon`